pkg/clusterautoscaler/validation.go

```diff
@@ -0,0 +1,27 @@
+package clusterautoscaler
+
+import (
+	"fmt"
+
+	cmv1 "github.com/openshift-online/ocm-sdk-go/clustersmgmt/v1"
+
+	"github.com/openshift/rosa/pkg/rosa"
+)
+
+const (
+	NoHCPAutoscalerSupportMessage = "Hosted Control Plane clusters do not support cluster-autoscaler configuration"
+	ClusterNotReadyMessage        = "Cluster '%s' is not yet ready. Current state is '%s'"
+)
+
+func IsAutoscalerSupported(runtime *rosa.Runtime, cluster *cmv1.Cluster) error {
+	if cluster.Hypershift().Enabled() {
+		return fmt.Errorf(NoHCPAutoscalerSupportMessage)
```

Comment:
Autoscaler is supported on HCP, but is only enabled for IBM if I'm not mistaken

Reply:
@gdbranco I replicated the logic from `create autoscaler` which doesn't seem to do any check to see if current org has capability to create autoscaler for their cluster. IIRC, IBM doesn't use the CLI, but OCM SDK directly?

Reply:
Correct, they are using SDK from what I know